paddle/fluid/framework/new_executor/instruction/instruction_util.cc

```diff
@@ -153,7 +170,10 @@ phi::DeviceContext* ParseDeviceContext(pir::Operation* op,
             op_name.compare(paddle::dialect::AllReduce_Op::name()) == 0 ||
             op_name.compare(paddle::dialect::Broadcast_Op::name()) == 0 ||
             op_name.compare(paddle::dialect::BroadcastOp::name()) == 0 ||
-            op_name.compare(paddle::dialect::AllGatherOp::name()) == 0) {
+            op_name.compare(paddle::dialect::AllGatherOp::name()) == 0 ||
+            op_name.compare(paddle::dialect::MpAllreduceSum_Op::name()) == 0 ||
+            op_name.compare(paddle::dialect::CIdentity_Op::name()) == 0 ||
+            op_name.compare(paddle::dialect::CConcatOp::name()) == 0) {
```

Comment:
Can we not add this if condition(L168-L176) anymore, as most operators have already been migrated.

Reply:
i have tried to remove them, but that will make infer error